src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -135,18 +136,17 @@ pub fn prebuilt_llvm_config(
         )
     });
 
-    let stamp = out_dir.join("llvm-finished-building");
-    let stamp = HashStamp::new(stamp, Some(smart_stamp_hash));
+    let stamp = BuildStamp::new(&out_dir).with_prefix("llvm").with_stamp(smart_stamp_hash);
```

Comment:
Stamp names mentioned in docs https://github.com/search?q=repo%3Arust-lang%2Frust+%22finished-building%22&type=code so need to do something with that.

Reply:
Follow-up PR is up: https://github.com/rust-lang/rustc-dev-guide/pull/2207